src/EventListener/InsertTagListener.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Terminal42\InsertTagsBundle\EventListener;
+
+use Contao\CoreBundle\DependencyInjection\Attribute\AsHook;
+use Psr\Log\LoggerInterface;
+use Terminal42\InsertTagsBundle\InsertTagHandler;
+
+class InsertTagListener
+{
+    public function __construct(
+        private readonly InsertTagHandler $handler,
+        private readonly LoggerInterface $logger,
+    ) {
+    }
+
+    #[AsHook('replaceInsertTags')]
+    public function onReplaceInsertTags(string $tag): string|false
```

Comment:
```suggestion
    public function __invoke(string $tag): string|false
```

And the attribute should be on the class instead.